packages/grid/x-data-grid/src/models/props/DataGridProps.ts

```diff
@@ -329,6 +330,12 @@ export interface DataGridPropsWithDefaultValues {
    * @default false
    */
   disableColumnResize: boolean;
+  /**
+   * The height of a row of grouping column headers.
+   * TODO: choose correctly this value
+   * @default 32
+   */
+  headerGroupingRowHeight: number;
```

Comment:
Could we reuse the header height for now? As I remember, the plan was even to [remove](https://github.com/mui/mui-x/issues/602) the `headerHeight` prop, so having a prop only for the column groups height is not very consistent.

Reply:
If I reuse `headerHeight` it will not be possible to have distinct height between groups and columns.

It was a proposal from a design perspective that having smaller group is interesting to reduce the distance between column headers and the group (easier to read relations then)

Reply:
From a visual design perspective, I'm used to column groups that have the same height. I would be curious to see if it's common to have different heights. I mean, maybe not making it possible to customize headerGroupingRowHeight:

```suggestion
```

would be a good thing. It's what we demo in https://mui.com/material-ui/react-table/#column-grouping is doing. It seems to be what all the examples in https://github.com/mui/mui-x/issues/195 do as well.

@gerdadesign I see that we have started a sketch in https://www.figma.com/file/W6TItYkiGJZjL5zLnfeSjy/Data-grid?node-id=0%3A1.

Reply:
> If I reuse headerHeight it will not be possible to have distinct height between groups and columns.

With CSS, and if we remove some inline styles, this will still be possible. The idea in #602 is to remove the `headerHeight` prop and use CSS to customize the header height. This prop only exists because the AutoSizer also wraps the header, so we need to subtract it.